src/blocks/theme/device/theme.device.ts

```diff
@@ -29,4 +31,5 @@ export const breakpointMap: Record<Breakpoint | 'initial', DeviceSizeName | ''>
   lp: 'laptop',
   ll: 'laptopL',
   dp: 'desktop',
+  uw: 'ultrawide',
```

Comment:
<img width="856" alt="Screenshot 2024-10-01 at 3 29 41 PM" src="https://github.com/user-attachments/assets/4221aae5-d7c9-4f01-8815-93c0d9605047">
You also need to update this function so Box also supports this size.

<Box display={{uw:"flex"}}></Box>